app/views/tutors/show.html.slim

```diff
@@ -0,0 +1,72 @@
+.row.mr-0.pt-2
+  .col-1
+  .col-10.p-2
+    h2.text-center.p-2
+      = t('.title')
+
+    .teacher-profile-block
+      .d-flex.flex-row
+        .d-flex.flex-column.col-2
+          .rounded-circle.p-3
+            = cl_image_tag(profile.avatar.medium)
+
+        .d-flex.flex-column.col-8
+          .d-flex.flex-row
+            .p-1.font-weight-bold
+              h4 = profile.full_name
+            .p-1.mt-1
+              - (1..5).each do
+                li.fa.fa-star
+
+          .p-1.font-weight-bold
+            = t('.subjects')
+            - profile.subjects.each do |subject|
+              = subject.name
+              | &nbsp;
+          .p-1.font-weight-bold
+            = t('.description')
+            = profile.about_me
+
+          .lesson-block
+            .d-flex.flex-row
+              .p-1.col-5
+                = t('.lessons_conducted')
+              .p-1.col-3
+                = t('.reviews')
+              .p-1.col-4
+                i.fa.fa-play-circle.px-2
+                = t('.video_introduction')
+
+            .d-flex.flex-row
+              .p-1.col-5
+                = t('.email')
+                = profile.email
+              .p-1.col-4
+                = t('.phone')
+                = profile.phone_number
+
+        .d-flex.flex-column.col-2.px-0
+          .button.btn.btn-success.btn-block
+            = t('.add_friend')
+          .button.btn.btn-warning.btn-block
+            i.fa.fa-envelope.px-2
+            = t('.send_email')
+          .button.btn.btn-warning.btn-block
+            i.fa.fa-phone.px-2
+            = t('.call')
```

Comment:
Ditto but telephone link
Should be `mail_to` link, and let's use font-awesome gem helper to add text, something like:
```
mail_to "tel:#{profile.phone_number}", class: 'btn btn-success btn-block' do
  = fa_icon :call, text:  t('.call')
```